packages/testing/src/api/index.ts

```diff
@@ -49,10 +56,11 @@ export function initializeAdminApp(options: any): admin.app.App {
   );
 }
 
-export function initializeTestApp(options: any): admin.app.App {
-  if (!('databaseName' in options)) {
-    throw new Error('databaseName not specified');
-  }
+export type TestAppOptions = {
+  databaseName: string;
+  auth: Object;
```

Comment:
Note that Object will only allow empty objects. You should use `object`.

https://blog.mariusschulz.com/2017/02/24/typescript-2-2-the-object-type